### PR TITLE
enable coverage test for github action

### PR DESCRIPTION
**Pull request by LHT129**

### CODE REVIEW

**Review by wxyucs — APPROVED** [review]
lgtm

File: .github/workflows/lcov.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+      - name: Install
+        run: |
+          python -m pip install --upgrade pip
+          pip install lcov_cobertura
```
Comment (jiaweizone) [inline .github/workflows/lcov.yml:24]: can install to VSAG docker image ?

File: scripts/check_cov.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+line_coverage=`lcov --summary coverage/coverage.info | grep "lines......" | awk '/lines/ { print $2 }' | cut -d '%' -f 1`
+line_coverage=$(printf "%.0f" $line_coverage)
+if [ "$line_coverage" -gt 84 ]; then
```
Comment (jiaweizone) [inline scripts/check_cov.sh:5]: line coverage can not set to 90% ?

Comment (LHT129) [reply scripts/check_cov.sh:5]: this is no used, codecov bot have method to set threshold, We will enable soon

File: .github/workflows/lcov.yml (hunk 1 above)
Comment (LHT129) [reply .github/workflows/lcov.yml:24]: good idea, but only install for coverage now

**Review by inabao — APPROVED** [review]
LGTM

File: .github/workflows/lcov.yml (hunk 1 above)
Comment (jiaweizone) [reply .github/workflows/lcov.yml:24]: OK

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**